plugin/pom.xml

```diff
@@ -86,6 +86,19 @@
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-text</artifactId>
       <version>1.9</version>
+      <exclusions>
```

Comment:
Makes sense to me. I've seen it as a dependency on other plugins.

Is there a quick way to tell how many plugins bundle commons-text today?

Reply:
normally 2 or more is enough to justify an api plugin.

Reply:
Excellent. I created [commons-text-api](https://github.com/sghill/commons-text-api) using commons-lang3-api as a model. I haven't introduced an open source plugin before, so I think someone with write access to jenkinsci will need to fork this?

Reply:
@sghill https://www.jenkins.io/doc/developer/publishing/requesting-hosting/
https://github.com/jenkins-infra/repository-permissions-updater

Reply:
@sghill I'd suggest keeping changelog out of the README and use GitHub releases as they integrate with Jenkins Plugin site.

Reply:
This may also be helpful: https://www.jenkins.io/doc/developer/publishing/releasing-cd/

Reply:
Thanks very much for the help (and patience!) @jetersen. This is now published and I have updated this PR to rely on commons-text-api, which in turn relies on commons-lang3-api.

Separately, I noticed commons-lang3-api has a nonstandard versioning scheme (3.12.0.0, rather than 3.12.0-v{count}.{sha}). Is that something we want to fix before rolling this out, or is it fine?

Reply:
Can be fixed at any point